dev-lang/erlang/erlang-21.0.4.ebuild

```diff
@@ -21,25 +21,25 @@ LICENSE="Apache-2.0"
 SLOT="0"
 KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~x86-fbsd ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~x86-macos ~x64-solaris"
 
-IUSE="dirty-schedulers doc emacs hipe java kpoll libressl odbc smp sctp ssl systemd tk wxwidgets"
-REQUIRED_USE="dirty-schedulers? ( smp )" #621610
+IUSE="doc emacs +hipe java +kpoll libressl odbc +pgo sctp ssl systemd tk +wxwidgets"
 
 RDEPEND="
+	emacs? ( virtual/emacs )
+	java? ( >=virtual/jdk-1.2:* )
+	odbc? ( dev-db/unixODBC )
+	sctp? ( net-misc/lksctp-tools )
 	ssl? (
 		!libressl? ( >=dev-libs/openssl-0.9.7d:0= )
 		libressl? ( dev-libs/libressl:0= )
 	)
-	emacs? ( virtual/emacs )
-	java? ( >=virtual/jdk-1.2:* )
-	odbc? ( dev-db/unixODBC )
+	sys-libs/ncurses:*
```

Comment:
`ncurses:*` is almost certainly wrong as slot `:5` does not provide headers to build on. Also, please move unconditional deps before USE conditionals.

Reply:
done